nengo/neurons.py

```diff
@@ -6,7 +6,7 @@
 from nengo.exceptions import SimulationError, ValidationError
 from nengo.params import DictParam, FrozenObject, NumberParam, Parameter
 from nengo.rc import rc
-from nengo.utils.numpy import is_array_like
+from nengo.utils.numpy import is_array_like, clip
```

Comment:
We like to keep the list of imported things alphabetized, unless there's some reason not to

Reply:
Done already in my WIP fixups